plugins/SkyCultureMaker/src/gui/ScmGeoLocGraphicsView.cpp

```diff
@@ -0,0 +1,386 @@
+/*
+ * Sky Culture Maker plug-in for Stellarium
+ *
+ * Copyright (C) 2025 Moritz Rätz
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "ScmGeoLocGraphicsView.hpp"
+#include <qevent.h>
+#include <qguiapplication.h>
+#include <qjsonarray.h>
+#include <QGraphicsSvgItem>
+#include <qscrollbar.h>
+#include <QFileDialog>
+
+#include <QJsonObject>
+#include <QJsonDocument>
+#include <QtMath>
+#include <cmath> // for M_PI
```

Comment:
`<QtMath>` is documented to define `M_PI` too. But why on Earth use `QtMath` at all? For `qPow`? How is it better than `std::pow`?

Besides the apparent uselessness, Qt is constantly making some crazy assumptions, like `qFloor` being templated (so that it could support large numbers) but returning a mere `int` (throwing away most of supportable integers), downcast from a call to `std::floor` (basically)!

So I'd advise to avoid `QtMath`, except maybe for the unique functions like the variadic `qHypot` overload or `qNextPowerOfTwo`, etc.